supp2u/src/components/Reviews/add/index.js

```diff
@@ -5,15 +5,16 @@ import AddReviewPresentation from "./AddReviewPresentation";
 import {toast} from "react-toastify";
 
 const mapPropsToState = ({customerReducer: {customer}}) =>({customer});
-export default connect(mapPropsToState, {addReview})(({addReview, history, customer})=>{
+export default connect(mapPropsToState, {addReview})(({addReview, history, customer, business_id})=>{
+
 
     const addReviewHandler =(review) =>{
-        const business_id =3; //TODO get business id from redux store or params
+        
         addReview({...review, business_id}, localStorage.customer_id)
             .then((res) => {
                 if(res.review){
                     toast.success(`Thank you for your feedback ${res.review}`);
-                    history.goBack();
+                    // history.goBack();
```

Comment:
This `if` block is easy to follow, and we've got the flexibility to adjust as the UI evolves.